chat/router.py

```diff
@@ -0,0 +1,110 @@
+from fastapi import APIRouter
```

Comment:
This is imported but unused.

Reply:
I modified the code in PR # 9. This is used in the new code and essential for fastapi part.